### DISCUSSION TITLE
Support ESP32 solo1 version

### DISCUSSION THREAD

**Discussion by dupan99** [benzino77/tasmocompiler#175] [Q&A]
Is there a way to integrate the ESP32 "solo1" version into the selection?

**Comment by benzino77** [comment]
If this is a standard (more less) ESP32 board you can just use `Generic ESP32` board. It is using `board = esp32dev` platformio setting.
You should ask a question about compatibility between Tasmota and Solo-1 board on Tasmota [discord channel ](https://discord.gg/Ks2Kzd4) or [Discussion](https://github.com/benzino77/tasmocompiler/discussions) section on Tasmota GitHub page. Quick glance shows me that it is a little bit stripped version of ESP32.

**Reply by benzino77** [reply]
It looks like Solo1 board is supported by Tasmota since 10 days!
Maybe I will find time next week to add it to TasmoComopiler.

**Reply by dupan99** [reply]
that would be great

**Reply by benzino77** [reply]
I have added support for Solo1 boards.
